.gitignore

```diff
@@ -2,3 +2,4 @@
 temp/
 fuzz
 /build/
+.DS_Store
```

Comment:
Tip: you can add files specific to your environment to `~/.gitignore_global`. These will apply to all your repos. :slightly_smiling_face:

Reply:
Ah, TIL! Thanks I've removed it